frontend-react/src/content/developer-resources/landing-page.mdx

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
can we rename this to developer-resources-index to keep consistent with the other landing pages? or rename the other ones?